### DISCUSSION TITLE
Nominating @andymoe as a maintainer of the content subteam

### DISCUSSION THREAD

**Discussion by fg-j** [paketo-buildpacks/feedback#102] [Content Team]
Andy has worked with me over the past few weeks on redesigning the website. He has deep context on the new site and will continue working extensively on it over the next several weeks/months. I'd like to promote him to a maintainer on the content team. Can I get 👍  / 👎   from @paketo-buildpacks/content-maintainers?

**Comment by ForestEckhardt** [comment]
@ryanmoran I believe that this is a supermajority of maintainers.

**Comment by ryanmoran** [comment]
Congrats @andymoe on your new maintainership!
